From 5364af83392188ab6c335464161917202b8cd799 Mon Sep 17 00:00:00 2001
From: Ofer Heifetz <oferh@marvell.com>
Date: Mon, 12 Dec 2016 09:42:34 +0200
Subject: [PATCH 1363/2241] arm64: clock: armada3700: add gating clock for
 eip97

    This patch adds gating clock support for Armada-3700 eip97
    by defining EIP97 gate clock in gate clock module.

Change-Id: I555550933934527a34f993f1d5ccd5de4313e2c7
Signed-off-by: Ofer Heifetz <oferh@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/34572
Tested-by: iSoC Platform CI <ykjenk@marvell.com>
Reviewed-by: Wilson Ding <dingwei@marvell.com>
---
 drivers/clk/mvebu/armada-3700.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/clk/mvebu/armada-3700.c b/drivers/clk/mvebu/armada-3700.c
index 9a3d0fb..53bc2c8 100644
--- a/drivers/clk/mvebu/armada-3700.c
+++ b/drivers/clk/mvebu/armada-3700.c
@@ -303,6 +303,7 @@ static const struct clk_gating_soc_desc armada_3700_north_bridge_gating_desc[] _
 	{ "spi-gate", "spi-core", 12, 0, CLK_GATE_SET_TO_DISABLE },
 	{ "twsi2", NULL, 16, 0, CLK_GATE_SET_TO_DISABLE },
 	{ "twsi1", NULL, 17, 0, CLK_GATE_SET_TO_DISABLE },
+	{ "eip97", NULL, 15, 0, CLK_GATE_SET_TO_DISABLE },
 	{ }
 };
 
-- 
2.7.4

